src/background/services/accounts/handlers/avalanche_getAddressesInRange.ts

```diff
@@ -147,11 +147,10 @@ export class AvalancheGetAddressesInRangeHandler extends DAppRequestHandler<
       });
 
       if (
-        await canSkipApproval(
-          request.site.domain,
-          request.site.tabId,
-          EXPOSED_DOMAINS
-        )
+        await canSkipApproval(request.site.domain, request.site.tabId, {
+          domainWhitelist: EXPOSED_DOMAINS,
+          allowInactiveTabs: true,
+        })
```

Comment:
Should we put all of the parameters to an object then?

Reply:
I can simplify in a follow up PR (so I don't need to collect approvals after updating again 😄)